crowdfunding_compassion/templates/myaccount_crowdfunding_page.xml

```diff
@@ -0,0 +1,55 @@
+<odoo>
+    <data>
+        <template id="myaccount_crowdfunding_view_template" name="MyAccount crowdfunding template">
+            <t t-call="website.layout">
+
+                <h2 class="blue my-2 center">Personal data</h2>
+                <section class="center">
+                    <t t-call="crowdfunding_compassion.myaccount_coordinates"/>
+                </section>
+
+                <h2 class="blue my-2 center">Your own projects</h2>
+                <section class="projects no-padding-top center">
+                    <t t-set="projects" t-value="owned_projects"/>
+                    <t t-call="crowdfunding_compassion.myaccount_project_board"/>
+                </section>
+
+                <h2 class="blue my-2 center">Projects you participate to</h2>
+                <section class="projects no-padding-top center">
+                    <t t-set="participations" t-value="participating_projects"/>
+                    <t t-call="crowdfunding_compassion.myaccount_participation_board"/>
+                </section>
+
+                <h2 class="blue my-2 center">Donations</h2>
+                <section class="center">
+                    <t t-set="donations" t-value="donations"/>
+                    <t t-call="crowdfunding_compassion.myaccount_donation_board"/>
+                </section>
+            </t>
+        </template>
+
+        <record id="myaccount_crowdfunding_view" model="ir.ui.view">
+            <field name="name">MyAccount view</field>
+            <field name="type">qweb</field>
+            <field name="key">crowdfunding.myaccount</field>
+            <field name="arch" type="xml">
+                <t t-call="crowdfunding_compassion.myaccount_crowdfunding_view_template"/>
+            </field>
+        </record>
```

Comment:
This is useless. The `<template>` tag is a shorthand for creating the `ir.ui.view` record.
```suggestion
```